app/view_models/auction_list_item.rb

```diff
@@ -43,92 +43,130 @@ def eligibility_label
     end
   end
 
-  def available?
-    auction_status.available?
+  def github_repo
+    auction.github_repo
   end
 
-  def bid_button_partial
-    if current_user_can_bid?
-      'auctions/bid_button'
+  def project_label_partial
+    if auction.github_repo
+      'auctions/project_label'
     else
       'components/null'
     end
   end
 
-  def item_footer_partial
-    if available?
-      'auctions/bids_and_form'
-    else
-      'auctions/winning_bid'
-    end
+  def project_label
+    auction.github_repo.sub('https://github.com/', '')
   end
 
   def winning_bid_partial
+    if over? && !auction.bids.any?
+      'auctions/no_bids'
+    elsif auction.type == 'single_bid'
+      winning_single_bid_partial
```

Comment:
by going to these other methods we are making the logic a bit more complicated -- can we put it all pack into one method? I know it' SUPER long, but it makes it easier to understand the branching